dotcom-rendering/src/footballMatches.ts

```diff
@@ -33,6 +33,7 @@ export type FootballMatch = MatchResult | MatchFixture | LiveMatch;
 
 type Competition = {
 	competitionId: string;
+	tag: string;
```

Comment:
In frontend this comes from the same place as the name and nation so presumably we will enrich with that data at the same stage (ie when we parse)?
https://github.com/guardian/frontend/blob/ca5f5086ce8eb21c1538a8698e809c941f61a6a7/sport/app/football/feed/Competitions.scala#L129

Reply:
Yes, it's part of the data being sent in https://github.com/guardian/frontend/pull/27655

For this to be a tag it needs the leading `football/`, so `football/fa-cup` or `football/serieafootball`. Therefore if you want to keep this field as "tag" I think you would need to adjust the URL generation in the component to handle this, and update the stories data to include the leading `football/`.

Note that the corresponding field being passed through in that PR is in path format, i.e. `/football/fa-cup` or `/football/serieafootball`, which we'll need to consider when parsing.